src/main/comments.js

```diff
@@ -342,6 +342,14 @@ function breakTies(tiesToBreak, text, options) {
     }
   });
 
+  for (const node of [precedingNode, followingNode]) {
+    if (node.comments && node.comments.length > 1) {
+      node.comments = node.comments.sort(
```

Comment:
```suggestion
      node.comments.sort(
```
`sort` muates the array.